.storybook/preview-head.html

```diff
@@ -1 +1 @@
-<link href="https://cdn.ons.gov.uk/sdc/design-system/37.0.2/css/main.css" rel="stylesheet" type="text/css">
\ No newline at end of file
+<link href="https://cdn.ons.gov.uk/sdc/design-system/60.0.1/css/main.css" rel="stylesheet" type="text/css">
```

Comment:
How does this get updated in consuming repos?

Reply:
I've take look and this does need to be updated in each repo, for an example see https://github.com/ONSdigital/blaise-management-information-reports/blob/main/public/index.html#L23